### DISCUSSION TITLE
Stargate generated in the open

### DISCUSSION THREAD

**Discussion by warderbrad** [Povstalec/StargateJourney#27] [General]
I am very excited about this mod. I have started a world with seed (5369984945557223422) and not far from spawn there was a stargate that was exposed. No idea where a temple is or anything. I did not see a cartouche to give the address to dial. What would someone do in such a situation?

**Comment by Povstalec** [comment]
Yup, that's the Stargate Pedestal with the original Earth gate (a.k.a. Beta Gate as they call it in the show). It doesn't have a Cartouche, since there wasn't one in the show either. You should be fine with just continuing on the intended quest of finding the buried Alpha Gate (as specified in [Getting Started)](https://github.com/Povstalec/StargateJourney/wiki/Getting-Started#finding-the-stargate) and taking the Beta Gate as a spare you can use.

There might be an issue where the map will lead to the Beta gate instead of the Alpha gate. Hopefully that doesn't happen, but if it does, just tell me here and I'll see if I can help somehow.

I plan on replacing the Beta gate pedestal with an underground Ancient Outpost so that it's more accurate and doesn't randomly pop up when you're randomly looking around.

**Reply by elvisinthehood** [reply]
The map led me to the Beta gate any ideas how to work around this?

**Reply by Povstalec** [reply]
I don't think I can make a safe way to find the Buried gate without potentially breaking some other things if not handled well, so I'd rather leave that as last resort.
Unless you've changed config to randomize addresses, the Abydos Address should be -26-6-14-31-11-29- so you can try dialing that.

If you have changed the config, you could either try having Cartouches with the Abydos address generate around the world.
I've got that datapack on a github repository, all that's needed is to download the code, extract the zip file and put the folder named Cartouche Monument Overworld Generation into your world's datapacks file.
https://github.com/Povstalec/StargateJourney-Datapacks

Or there is of course the potentially dangerous method, although I think even if something broke it would still be possible to fix it.

**Reply by elvisinthehood** [reply]
That did the trick thanks
